gen_cert.py

```diff
@@ -1841,8 +1834,32 @@ def fontlist_with_style(a_style):
         yOffset = minYOffset + ((max_height - height) / 2)
         paragraph.drawOn(PAGE, GUTTER_WIDTH - (name_style.fontSize / 12), yOffset)
 
+
+        # Parse Translations:
+        translated_success_text = ""
+        translated_interstitial_intro = ""
+        translated_disclaimer = ""
+        translated_verify_text = ""
+
+        if self.default_translations and self.locale in self.default_translations:
```

Comment:
We should just reference `settings.DEFAULT_TRANSLATIONS`, but beyond that, the default translations dict should _never_ be empty, so we shouldn't gate on that here. Instead, we should allow it to break if someone mistakenly deletes the defaults.

```python
if self.locale in settings.DEFAULT_TRANSLATIONS:
    # do stuff
```